### PR TITLE
VUE-199 KvRangeSlider

### PR DESCRIPTION
**Pull request by ryan-ludwig**
- Create a presentational component based on the slider in the lend filter page.
- Allow v-model or :value binding
- Ensure a label element is used and that the value of the slider is not part of the label.
- Update usage in the site

### CODE REVIEW

File: src/components/Kv/KvRangeSlider.vue [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+		}
+	},
+	methods: {
+		onInput($event) {
+			this.$emit('input', parseFloat($event.target.value, 10));
+		},
+		onChange($event) {
+			this.$emit('change', parseFloat($event.target.value, 10));
+		},
+		sliderStyle() {
```
Comment (eddieferrer) [inline src/components/Kv/KvRangeSlider.vue]: This should be a computed property instead of a method.

**Comment by ryan-ludwig** [comment]
@BoulderBrains Gave the CSS a refactor based on lessons [learned here](https://css-tricks.com/sliding-nightmare-understanding-range-input/,) 🤯 we're up to around 20px hit target area now. Still not amazing for touch, but better than 6px. Tested on Firefox, Chrome, Safari. Downloading XCode for iOS emulation now.

**Comment by mcstover** [comment]
@ryan-ludwig is our scss compilation going to pick up and make the css variables compatible with IE?

**Comment by ryan-ludwig** [comment]
@mcstover Good question. No, it won't make them compatible, but in this instance IE isn't using them. Webkit and blink are using that fancy CSS var gradient, while IE is using it's own "::-ms-fill-lower " pseudo element that uses a regular background color.

**Pull request merged**